controller/workitem_blackbox_test.go

```diff
@@ -843,12 +842,11 @@ func newChildIteration(ctx context.Context, db *gorm.DB, parentIteration *iterat
 	iterationRepo := iteration.NewIterationRepository(db)
 
 	parentPath := append(parentIteration.Path, parentIteration.ID)
-	userActive := false
 	itr := iteration.Iteration{
 		Name:       "Sprint 101",
 		SpaceID:    parentIteration.SpaceID,
 		Path:       parentPath,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c